### ISSUE TITLE
Watermark in corner of logo

### ISSUE THREAD

**Issue by ConvexPolygon** [Scratch-Client-4/landing#12]
I think the @Archon + @Ekmand logos shouldn't be in the corner, just @Ekmand's or none.
This would only be fair because @Ekmand imagined & Designed it, and all of us are helping. However, on the team page, it should say 'Founded by Ekmand and Archon ...'

**Comment by micahlt** [comment]
@ConvexPolygon You're right- but the current SC4 logo wasn't intended to be final.  It's boring and not unique.  I just haven't come up with a good logo yet.

**Comment by ConvexPolygon** [comment]
Ok. That's what thought, any ideas for the new logo
@Scratch-Client-4/everyone

**Comment by githubfan** [comment]
we need an icon idea

and we kinda also need another way of communication, GitHub/gitter kinda sucks

**Comment by micahlt** [comment]
@githubfan What's wrong with Gitter?

**Comment by githubfan** [comment]
its kinda... meh
I'd be happy to use discord/teams or anything else tho

**Comment by ConvexPolygon** [comment]
@githubfan Microsoft Teams? No.

**Comment by rudrathegreat** [comment]
yes I am in for teams

**Comment by githubfan** [comment]
alright @micahlt  do you wanna make it?

**Comment by micahlt** [comment]
👍 Please list your Microsoft Teams usernames.

**Comment by rudrathegreat** [comment]
rudratheastronaut@gmail.com

**Comment by githubfan** [comment]
think I'm codingmaster24@aol.com

**Comment by ConvexPolygon** [comment]
me@jtechnologies.xyz

**Comment by locness3** [comment]
My account has my real name, and it needs a first and last name (so I can't use just Locness), so idk if i'll join

**Comment by githubfan** [comment]
whats so private about your real name? just use locness three as your name

**Comment by locness3** [comment]
litterally "three"? why not

**Comment by locness3** [comment]
alright, locness3@outlook.com

**Comment by YOYITsM3M8** [comment]
i dont have microsoft teams sooooooooo

**Comment by YOYITsM3M8** [comment]
dont even have a windows pc

**Comment by Mai-Trys** [comment]
Dang, I don’t have Microsoft teams. I have Discord, but I will see if I can use Microsoft teams with my Pixlebook which is a Chrome device.

**Comment by ConvexPolygon** [comment]
> Dang, I don’t have Microsoft teams. I have Discord, but I will see if I can use Microsoft teams with my Pixlebook which is a Chrome device.

Microsoft Teams is an online application. I can run it on my desktop windows, ubuntu, on my chromebook, and on a macbook.

**Comment by BonfireScratch** [comment]
> > Dang, I don’t have Microsoft teams. I have Discord, but I will see if I can use Microsoft teams with my Pixlebook which is a Chrome device.
> 
> Microsoft Teams is an online application. I can run it on my desktop windows, ubuntu, on my chromebook, and on a macbook.

That's an awful lot of devices!

**Comment by githubfan** [comment]
Yup

**Comment by ConvexPolygon** [comment]
> > > Dang, I don’t have Microsoft teams. I have Discord, but I will see if I can use Microsoft teams with my Pixlebook which is a Chrome device.
> > 
> > 
> > Microsoft Teams is an online application. I can run it on my desktop windows, ubuntu, on my chromebook, and on a macbook.
> 
> That's an awful lot of devices!

They aren't all mine.
in reality, I have my chromebook and my desktop. The Mac is my parent's

**Comment by Ekmand** [comment]
> Dang, I don’t have Microsoft teams. I have Discord, but I will see if I can use Microsoft teams with my Pixlebook which is a Chrome device.

For discord my tag is Ekmand#1802 , and microsoft teams is a web app, so it should work on all devices.

**Comment by Mai-Trys** [comment]
@Ekmand Thank you.

**Comment by rudrathegreat** [comment]
this is pretty stupid, but I can't seem to join the team on Teams (pun not intended). Figuring out the solution, needs to be solved ASAP for me to provide updates on a regular basis

**Comment by githubfan** [comment]
Yea. If anyone wants to msg me on discord, githubfan#7548

**Comment by rudrathegreat** [comment]
i'm into microsoft teams! btw are we pulling all branches to master and then deploying the master branch, because currently we hav a different branch deployed, for consistency, let use one branch where we pull other branches to and then deploy that branch, instead of switching between multiple branches

**Comment by BonfireScratch** [comment]
I could design a logo for you. @micahlt @rudrathegreat @locness3

**Comment by micahlt** [comment]
@BonfireScratch Feel free to make a draft!  I'm working on one as well.

**Comment by rudrathegreat** [comment]
I think I'll be fine if I stick with my own logo on Scratch! thanks for the offer though @BonfireScratch

**Comment by BonfireScratch** [comment]
I mean for the website @rudrathegreat

**Comment by rudrathegreat** [comment]
oh, then that's fine, i often get things mixed up! XP

**Comment by githubfan** [comment]
lol. anyone on discord rn?

**Comment by rudrathegreat** [comment]
no not yet. btw I somehow don't have write access can some give me that access pls, it's decimating my work output!

**Comment by YOYITsM3M8** [comment]
wait, is microsoft teams part of offics 365?
bc i dont have that

**Comment by BonfireScratch** [comment]
For the logo, should it be based on the scratch logo? Or should it be more abstract?

**Comment by micahlt** [comment]
@rudrathegreat That was intentional - just in case, the permissions are now configured where @Hexsphere and I have to review and merge pulls.  

@BonfireScratch Probably more abstract.

**Comment by BonfireScratch** [comment]
@micahlt hmm... Would you like more linear and geometric shapes (triangles, squares e.t.c) or more circles and curves in the logo?

**Comment by micahlt** [comment]
@micahlt That's your decision!  I'm also working on a concept.

**Comment by BonfireScratch** [comment]
Looks good to me

**Comment by githubfan** [comment]
I like it!

**Comment by rudrathegreat** [comment]
like it!

**Comment by micahlt** [comment]
@Ekmand What do you think?  This was originally your project xD

**Comment by ConvexPolygon** [comment]
I like that design :)

**Comment by Mai-Trys** [comment]
It's nice, but I think it's a little bold. Maybe instead of red, we use blue?

**Comment by rudrathegreat** [comment]
Can anyone please check the pull request I opened. I need feedback ASAP

**Comment by MiniCoder11** [comment]
I really like the concept but I also agree with Mai-Trys. I’m not a lead-designer in this project but I still think that the red-on-black lettering doesn’t quite match the UI style that is used throughout the rest of the interface. Maybe changing the lettering to a lighter blue would match?

**Comment by rudrathegreat** [comment]
yeah I agree with @MiniCoder11. The design is nice but the colors don't match, maybe the blue colour with a slightly less darker grey background should do the trick!

**Comment by micahlt** [comment]
@MiniCoder11 @rudrathegreat All right, I'll work on that.

**Comment by YOYITsM3M8** [comment]
@micahlt I like it

**Comment by ConvexPolygon** [comment]
> yeah I agree with @MiniCoder11. The design is nice but the colors don't match, maybe the blue colour with a slightly less darker grey background should do the trick!

I agree. It will look nice and bring part of the Scratch-y feel into the logo.

**Comment by locness3** [comment]
I've marked the discussion about Microsoft Teams as off-topic and collapsed it.
